app/controllers/reativo/crud_controller.rb

```diff
@@ -8,9 +8,9 @@ def theme_cell
       end
 
       def _run_options(options)
-        options.merge( "current_user" => self.try(:current_user) )
```

Comment:
thinking about this change.. the standard will be trailblazer 2.1, all other projects using trb 2.0 will need to overwrite the `_run_options` method to keep the current behaviour